### PR TITLE
Feat(eos_designs): sflow_polling_interval

### LINKED ISSUE #4813: Feat(eos_designs): Add polling interval knob for sflow_settings in  eos_designs
### Enhancement summary

The option to modify the polling interval for sflow_settings does not exist.

le01a(config)#sflow polling-interval ?
  <1-3600>  Seconds

### Which component of AVD is impacted

eos_designs

### Use case example

People would like to edit the sflow polling interval from the default 2 seconds.

### Describe the solution you would like

The goal would be to add the following to the sflow_settings data model as shown below:

```yaml
sflow_settings:
  polling_interval: 1
  sample: 
    rate: 16384
  destinations:
    - destination: 127.0.0.1
      vrf: use_mgmt_interface_vrf
```

### Describe alternatives you have considered

I am ready to submit the PR.

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ernestoherrerab**
## Change Summary

Adds the option to modify the polling interval for sflow_settings.

## Related Issue(s)

Fixes #4813 

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
- Updated sflow_settings schema 
- Updated SflowSettings class 
- Updated eos_designs documentation

```yaml
sflow_settings:
  polling_interval: 1
```
## How to test
Tested with molecule and on a vEOS lab.

### Repository Checklist

- [ X] My code has been rebased from devel before I start
- [ X] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ X] My change requires a change to the documentation and documentation have been updated accordingly.
- [ X] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ keys:
       The sFlow process will only be configured if any interface is enabled for sFlow.
       For default enabling of sFlow for various interface types across the fabric see `fabric_sflow`.
     keys:
+      polling_interval:
+        type: int
+        description: Interval for sending counter data to the sFlow collector
```
Comment (gmuloc) [inline python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml]: For int we always add convert_types from str
```suggestion
        convert_types:
          - str
        description: Interval for sending counter data to the sFlow collector.
```

File: python-avd/pyavd/_eos_designs/structured_config/flows/__init__.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def sflow(self) -> dict | None:
 
         # At this point we have at least one interface with sFlow enabled
         # and at least one destination.
-        sflow = {"run": True, "sample": self.inputs.sflow_settings.sample.rate}
+        sflow = {"run": True,"polling_interval": self.inputs.sflow_settings.polling_interval ,"sample": self.inputs.sflow_settings.sample.rate}
```
Comment (gmuloc) [inline python-avd/pyavd/_eos_designs/structured_config/flows/__init__.py]:
```suggestion
        sflow = {"run": True, "polling_interval": self.inputs.sflow_settings.polling_interval , "sample": self.inputs.sflow_settings.sample.rate}
```

**Review by gmuloc — CHANGES_REQUESTED** [review]

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml (hunk 1 above)
Comment (ernestoherrerab) [reply python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml]: Updated

File: python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml [hunk 3]
```diff
@@ -15,6 +15,11 @@ keys:
       The sFlow process will only be configured if any interface is enabled for sFlow.
       For default enabling of sFlow for various interface types across the fabric see `fabric_sflow`.
     keys:
+      polling_interval:
+        type: int
+        convert_types:
+          - str
+        description: Interval for sending counter data to the sFlow collector.
```
Comment (gmuloc) [inline python-avd/pyavd/_eos_designs/schema/schema_fragments/sflow_settings.schema.yml]:
```suggestion
        description: Interval in seconds for sending counter data to the sFlow collector.
```

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Review by gmuloc — APPROVED** [review]

**Pull request merged**